packages/pyml/pyml.20171102/opam

```diff
@@ -0,0 +1,14 @@
+opam-version: "1.2"
+name: "pyml"
+maintainer: "Thierry Martinez <martinez@nsup.org>"
+authors: "Thierry Martinez <martinez@nsup.org>"
+homepage: "http://pyml.gforge.inria.fr"
+bug-reports: "http://pyml.gforge.inria.fr/tracker"
+license: "BSD"
+dev-repo: "https://scm.gforge.inria.fr/anonscm/git/pyml/pyml.git"
+build: [make "all" "pymltop" "pymlutop" {utop:installed} "PREFIX=%{prefix}%"]
+install: [make "install" "PREFIX=%{prefix}%"]
+remove: [make "uninstall" "PREFIX=%{prefix}%"]
+depends: ["ocamlfind" {build}]
+depopts: ["utop"]
+version: "20171102"
```

Comment:
This needs a line : `available: [ ocaml-version >= "4.02.0" ]` and it should be good merge